install/kubernetes/cilium/templates/cilium-agent/clusterrole.yaml

```diff
@@ -102,6 +102,9 @@ rules:
   - ciliumlocalredirectpolicies/finalizers
   - ciliumegressnatpolicies
   - ciliumendpointslices
+  - ciliumenvoyconfigs
+  - ciliumenvoyconfigs/status
+  - ciliumenvoyconfigs/finalizers
```

Comment:
Do you need finalizers?

Reply:
Ditto for other clusterroles, if you're not using these, please remove them.

Reply:
Removed /status and /finalizers as neither is (currenly) used.